alf/examples/merlin_dmlab_collect_good_objects.gin

```diff
@@ -0,0 +1,77 @@
+# MERLIN on deepmindlab rooms_collect_good_objects
+
+import alf.algorithms.merlin_algorithm
+import alf.algorithms.decoding_algorithm
+import alf.trainers.on_policy_trainer
+
+# `DeepmindLab` are required,
+#   see `https://github.com/deepmind/lab` to build `DeepmindLab`
+
+# environment config
+import alf.environments.suite_dmlab
+import alf.environments.wrappers
+
+# environment config
+create_environment.env_load_fn=@suite_dmlab.load
+create_environment.env_name="contributed/dmlab30/rooms_collect_good_objects_train"
```

Comment:
What is the performance on  rooms_collect_good_objects_test?
For some tasks, we need to be able to specify different env for evaluation.

Reply:
evaluate for a intermediate checkpoint ( evaluate episode returns 7-8 ),
it seems good 

![rooms_collect_good_objects_train](https://user-images.githubusercontent.com/1267359/70106905-e4974d00-167f-11ea-9899-d09e451ebc6f.gif)

![rooms_collect_good_objects_test](https://user-images.githubusercontent.com/1267359/70106907-e7923d80-167f-11ea-93a8-dfeb81e5194b.gif)

Reply:
> For some tasks, we need to be able to specify different env for evaluation.

Let's do this at another pr

Reply:
> evaluate for a intermediate checkpoint ( evaluate episode returns 7-8 ),
> it seems good
Do you use play to do this. But for play, the batch size is 1, without training=False, BatchNormalization might be bad?

Reply:
> Do you use play to do this.

Yes, and the checkpoint is on about 5M env step, performance is not good yet , 
 also played after converged (15M), result is much better, average episode returns about 9.4   

> But for play, the batch size is 1, without training=False, BatchNormalization might be bad?

i am not sure about this